### PR TITLE
Change colors -> colours and BN's -> BNs

### PR DESCRIPTION
**Pull request by Xinnoh**
I think I got it right this time

### CODE REVIEW

File: wiki/Article_Styling_Criteria/en.md [hunk 1]
```diff
@@ -472,7 +472,7 @@ Use `e.g.` when abbreviating "for example" and use `i.e.` for "that is". Side no
 
 ---
 
-When pluralizing acronyms and/or abbreviations, use an apostrophe followed by the letter `s` (`'s`) (e.g. `BN's`, `QAT's`, `GMT's`). If this writing style appears foreign or weird to you, try rewriting the sentence to avoid doing so in the first place.
+When pluralizing acronyms and/or abbreviations, do not use an apostrophe followed by the letter `s` (`'s`) (e.g. `BNs`, `QATs`, `GMTs`).
```
Comment (MegaApplePi) [inline wiki/Article_Styling_Criteria/en.md]: Change to:
```
Do not pluralize acronyms and/or abbreviations.
```

Comment (Ephemeralis) [reply wiki/Article_Styling_Criteria/en.md]: Probably the best approach, yeah. 'the BN, the GMT, the QAT' works better

Comment (Xinnoh) [reply wiki/Article_Styling_Criteria/en.md]: missed this, fixed

File: wiki/Article_Styling_Criteria/en.md [hunk 2]
```diff
@@ -472,7 +472,7 @@ Use `e.g.` when abbreviating "for example" and use `i.e.` for "that is". Side no
 
 ---
 
-When pluralizing acronyms and/or abbreviations, use an apostrophe followed by the letter `s` (`'s`) (e.g. `BN's`, `QAT's`, `GMT's`). If this writing style appears foreign or weird to you, try rewriting the sentence to avoid doing so in the first place.
```
Comment (TPGPL) [inline wiki/Article_Styling_Criteria/en.md]: or -> nor 
also double spaces after `or`

File: wiki/Beatmap_Editor/Design/ja.md [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
     画像のサイズは800x600ピクセルを基本的に超えてはいけません。
 
-4.  あなたが画像に対して加えたいコマンド(移動に関するMove, サイズに関するScele, 透明度に関するFade, 回転に関するRotate, 色に関するColor)を選び、エフェクトを入れたいタイミングまで移動したら、タイムラインの下にある"+"を押し、エフェクトのための"キーフレーム"を入れてください。
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Design/ja.md]: its a command so dont change?

File: wiki/Beatmap_Editor/Song_Setup/zh.md [hunk 4]
```diff
@@ -160,7 +160,7 @@ Approach Rate通常缩写为"**AR**"。
 
 ### 物件颜色(Hitcircle/Slider Combos)
 
-这些选项可以直接规定某一串combo的颜色。对于一个谱面来说，combo的颜色是很重要的,一个铺面由很多滑条和圆圈组成，合适的颜色配置能提升谱面的视觉效果。 在页面上点击一个Combo将会打开你电脑上的取色板，然后你就可以选择对应的颜色了，记住这个颜色要适合和你的背景图像。你最多可以选择8个自定义颜色。点击“Remove Combo Color”会移除最后一个自定义颜色。
+这些选项可以直接规定某一串combo的颜色。对于一个谱面来说，combo的颜色是很重要的,一个铺面由很多滑条和圆圈组成，合适的颜色配置能提升谱面的视觉效果。 在页面上点击一个Combo将会打开你电脑上的取色板，然后你就可以选择对应的颜色了，记住这个颜色要适合和你的背景图像。你最多可以选择8个自定义颜色。点击“Remove Combo Colour”会移除最后一个自定义颜色。
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Song_Setup/zh.md]: its that way is client i think so dont change

File: wiki/People/osu!_Alumni/it.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 osu! Alumni
 =============
 
-Gli **osu! Alumni** sono coloro che sono conosciuti per i loro contributi che se ne sono andati. Quando avremo le risorse ereggeremo una statua per ciascun membro nella piazza della città. [Link](http://osu.ppy.sh/g/16)
+Gli **osu! Alumni** sono o che sono conosciuti per i loro contributi che se ne sono andati. Quando avremo le risorse ereggeremo una statua per ciascun membro nella piazza della città. [Link](http://osu.ppy.sh/g/16)
```
Comment (TPGPL) [inline wiki/People/osu!_Alumni/it.md]: ??????????????????????? 🤔

**Comment by Ephemeralis** [comment]
@Xinnoh address reviews plz

File: wiki/Beatmap_Editor/Song_Setup/zh.md (hunk 4 above)
Comment (Xinnoh) [reply wiki/Beatmap_Editor/Song_Setup/zh.md]: I can't edit this file for some reason, github keeps breaking

Comment (Xinnoh) [reply wiki/Beatmap_Editor/Song_Setup/zh.md]: turns out google was auto translating the page -_-

**Comment by Xinnoh** [comment]
@Ephemeralis addressed

**Comment by Ephemeralis** [comment]
learn to use single commits reeeeeeeeeeeeeeeeee

**Pull request merged**